generator/src/test/java/com/scottlogic/deg/generator/cucumber/features/operators/string/ContainingRegex.feature

```diff
@@ -206,27 +206,27 @@ Scenario: Running a 'containingRegex' request for a range over a single characte
 
 Scenario: Running a 'containingRegex' for a maximum length smaller than the minimum length should fail with an error
      Given foo is containing regex /[a]{1,0}/
-     Then I am presented with an error message
+     Then the profile is invalid because "Illegal repetition range near index 7\r?\n\[a\]\{1,0\}\r?\n       \^"
```

Comment:
`\r` is optional here, as when the tests run on a *nix computer (e.g. Circle CI), `\r` isn't emitted, however it is on Windows.